docs/source/developer/xkcd_extension_tutorial.rst

```diff
@@ -0,0 +1,4 @@
+The extension tutorial has moved!
```

Comment:
```suggestion
:orphan:

The extension tutorial has moved!
```

Reply:
Today I learned about the `:orphan:` metadata tag in sphinx that makes the test failures go away.

Reply:
@krinsman - can you add the `:orphan:` metadata at the top of the file so the docs build doesn't give an error about the file not being in the table of contents?

Reply:
@krinsman - we're trying to release 1.0.3 in about 14 hours. If you have time to accept the above change, that would be great. If you don't have time to add this metadata, no worries, we can merge this and add it later (I can't push to your branch to add it to this PR).

Reply:
Thanks for the information about the `:orphan:` metadata tag! This is a good way to learn about using Sphinx which is useful anywhere. Anyway it's definitely a good idea to add that tag to this page.

Sorry about my slow response; I need to change my GitHub settings to also send notifications to the e-mail for my summer job.

Reply:
It was perfect, no worries. Thanks again for adding this!